runtime/lua/man.lua

```diff
@@ -8,6 +8,12 @@ local M = {}
 --- @param env? table<string,string|number>
 --- @return string
 local function system(cmd, silent, env)
+  if vim.fn.executable(cmd[1]) == 0 then
+    error(string.format("not found: '%s'", cmd[1]), 0)
+  elseif vim.uv.fs_access(cmd[1], 'X') then
+    error(string.format("no permission to execute: '%s'", cmd[1]), 0)
```

Comment:
the example code i gave intentionally used double quotes in the message.
```suggestion
    error(string.format('not found: "%s"', cmd[1]), 0)
  elseif vim.uv.fs_access(cmd[1], 'X') then
    error(string.format('not executable: "%s"', cmd[1]), 0)
```